docs/main/02-intro.md

```diff
@@ -1,71 +1,131 @@
 ---
 title: Docs Introduction
-description: Introduction to Kratix and how to get started quickly. Includes guides on how to read Kratix docs and pointers to other pages.
+description: Introduction to Kratix.
 keywords: [kratix documentation]
 sidebar_label: Introduction
 slug: /
 ---
 
-# Introduction
-
-Kratix is the open-source Platform Framework, that enables you to provide anything as-a-Service, while incorporating your key business processes and managing everything as a fleet.
-
 :::info
-Ready to get started? Follow our [Quick Start](01-quick-start.md) to jump right in.
+Looking for an **Enterprise solution**? Check out [Syntasso Kratix
+Enterprise (SKE)](https://syntasso.io/pricing)
 :::
 
-## Why Kratix?
+# Introduction to Kratix
+
+Kratix is an open-source platform framework built by
+[Syntasso](https://syntasso.io) to help you create an internal developer
+platform exactly the way your organisation needs it. We believe great platforms
+can't be bought off the shelf, they must be built and tailed to your
+organisations needs. Kratix provides the flexibility, integration, and
+automation needed to build your platform your way.
+
+Some teams describe Kratix as a platform orchestrator, designed specifically to
+help platform teams deliver more value, faster, safer, and with less overhead.
```

Comment:
```suggestion
Kratix is an example platform orchestrator in the [Thoughtworks Tech Radar](https://www.thoughtworks.com/en-gb/radar/techniques/platform-orchestration) and is designed specifically to
help platform teams deliver more value, faster, safer, and with less overhead.
```

But most specifically, not using "some teams", we should make sure to own the language on this page